files/initramfs-tools/varlog

```diff
@@ -17,6 +17,12 @@ for x in "$@"; do
     case "$x" in
         varlog_size=*)
             varlog_size="${x#varlog_size=}"
+            ;;
+        resize2fs-host)
```

Comment:
where do you pass this data into /proc/cmdline?

Reply:
the resize2fs-host is pass in the bootline in the grub config file that is used for migration : nos_to_sonic_grub.cfg. e.g:

linux   /image-master.0-dirty-20180129.024732/boot/vmlinuz-3.16.0-4-amd64 root=/dev/sda8 rw console=tty0 console=ttyS1,9600n8 quiet loop=image-master.0-dirty-20180129.024732/fs.squashfs loopfstype=squashfs apparmor=1 security=apparmor varlog_size=4096 fast-reboot resize2fs-host